src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -2961,6 +2971,139 @@ NEVER_INLINE void InterpreterSlowPath::initializeGlobalVariable(ExecutionState&
     ASSERT_NOT_REACHED();
 }
 
+NEVER_INLINE void InterpreterSlowPath::createObjectOperation(ExecutionState& state, CreateObject* code, ByteCodeBlock* byteCodeBlock, Value* registerFile)
+{
+    if (code->m_dataRegisterIndex != REGISTER_LIMIT) {
+        CreateObjectPrepare::CreateObjectData* data = reinterpret_cast<CreateObjectPrepare::CreateObjectData*>(registerFile[code->m_dataRegisterIndex].payload());
+        Object* obj = registerFile[code->m_registerIndex].asObject();
+        ObjectStructureItemTightVector properties;
+        properties.reset(data->m_properties.takeBuffer(), data->m_values.size());
+        obj->m_structure = ObjectStructure::create(state.context(), std::move(properties));
```

Comment:
Instead of allocating a temporal `ObjectStructureItemTightVector(properties)` to copy the `data->m_properties` here, 
What about defining `data->m_properties` as `ObjectStructureItemTightVector` to just move the data directly?
Is this not possible because properties of the same name could be initialized together ?

Reply:
ObjectExpression not always have immutable count of properties. ex) spread element, duplicated key
But TightVector needs GC_MALLOC every expand of property
That's why I use Vector for data->m_properties